hangupsbot/plugins/default.py

```diff
@@ -48,7 +48,18 @@ def echo(bot, event, *args):
 
         yield from command.run(bot, event, *["convecho", "id:" + convid, _text])
 
-
+def admins(bot,event,*args):
+    admin_list=bot.get_config_option('admins')
```

Comment:
Does it really need a docstring? Happy to put one it, but...it just outputs admins, there's no values or arguments to it.